src/tlo/simulation.py

```diff
@@ -298,9 +300,17 @@ def initialise(self, *, end_date: Date) -> None:
             raise SimulationPreviouslyInitialisedError(msg)
         self.date = self.start_date
         self.end_date = end_date  # store the end_date so that others can reference it
+
         for module in self.modules.values():
             module.initialise_simulation(self)
         self._initialised = True
+        
+        # Since CollectEventChains listeners are added to notified upon module initialisation,
+        # this can only be dispatched here.
+        # Otherwise, would have to add listener outside of CollectEventChains initialisation
+        
+        # Dispatch notification that pop has been initialised
+        notifier.dispatch("simulation.pop_has_been_initialised", data={"EventName" : "StartOfSimulation"})
```

Comment:
Make notification key `Simulation.post-initialise`. You don't need to return any data here. The registered listener should know it's being triggered by the notification it's listening for.